src/SearchField/SearchField.scss

```diff
@@ -10,8 +10,16 @@
     pointer-events: none;
```

Comment:
<img width="969" alt="image" src="https://user-images.githubusercontent.com/2828721/172993962-3c31b194-af6f-4b90-9185-89f1e3cd46f9.png">

<img width="986" alt="image" src="https://user-images.githubusercontent.com/2828721/172994033-69ba976f-de69-4bf0-b30e-f5c18531937b.png">

Should the focusable search/clear buttons have an appropriate focus style as well? The current outline appears to be native browser outline.

Reply:
Added this focusable variant =>
<img width="982" alt="image" src="https://user-images.githubusercontent.com/93188219/173899110-2c2b1736-f824-42e3-97c3-f23dc94b3e4a.png">
<img width="962" alt="image" src="https://user-images.githubusercontent.com/93188219/173899319-74094b18-17a0-418f-aaae-bf80c16c65e6.png">
I think looks good) What you think for this variant?

Reply:
LGTM!